Remove unstable auth methods

nodev11 opened the pull request:
Maybe now is a good time to remove the old auth methods.

nodev11 committed:
Update alexa_remote_control.sh

Removed unstable authentication methods.

nodev11 committed:
Merge pull request #1 from nodev11/nodev11-patch-1

Update alexa_remote_control.sh

nodev11 committed:
Update README.md

Remove text related to old auth methods.

nodev11 committed:
Merge pull request #2 from nodev11/nodev11-patch-1

Update README.md

adn77 commented:
Sorry for taking so long to reply.
There are quite a few people out there who're still using the MFA enabled login method (and it might still be working for them). I appreciate your contribution but I wouldn't want to introduce such a breaking change just yet.

Alex

nodev11 committed:
Update alexa_remote_control.sh

Initial login page has changed #166 (rudybrian):

https://github.com/thorsten-gehrig/alexa-remote-control/issues/166

nodev11 committed:
Update alexa_remote_control.sh

Add changelog entry

nodev11 committed:
Update alexa_remote_control.sh

removed function toEpoch()

nodev11 committed:
Delete alexa_remote_control_plain.sh

Remove outdated alexa_remote_control_plain.sh

nodev11 committed:
Update alexa_remote_control.sh

Clearified comment

nodev11 committed:
Update alexa_remote_control.sh

Remove anything related to oathtool

nodev11 committed:
Update alexa_remote_control.sh

Removed/corrected two comments

nodev11 committed:
Update README.md

nodev11 committed:
Update alexa_remote_control.sh

Updated the functions last_alexa() and last_command() (thanks to Ralf Mueller).
Modified some 'date' commands to work on embedded devices.

nodev11 committed:
Update alexa_remote_control.sh

Minor fixes.

nodev11 committed:
Update alexa_remote_control.sh

Minor fix.

nodev11 committed:
Minor changes
